pkg/sql/colexec/external/external.go

```diff
@@ -647,3 +353,342 @@ func ScanFileData(param *ExternalParam, proc *process.Process) (*batch.Batch, er
 	bat.Cnt = 1
 	return bat, nil
 }
+
+func getData(bat *batch.Batch, Line []string, rowIdx int, param *ExternalParam) error {
```

Comment:
OK, so this getData, is rather expensive.   Is it possible to vectorvise it?   That is, for each batch, each column, do one switch type, then finish this column.

Reply:
I will try